TestApps/TestAppKeyboard/TestAppKeyboard.csproj

```diff
@@ -12,6 +12,7 @@
   <ItemGroup>
     <PackageReference Include="GitVersion.MsBuild" Version="5.6.8" PrivateAssets="all" />
     <PackageReference Include="NDesk.DBus" Version="0.15.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
```

Comment:
I don't think this project needs it. (Notice there is no Nunit package reference)